dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/PutJob.kt

```diff
@@ -107,7 +107,10 @@ class PutJob(private val putJobData: JobData) : JobService(), PrepStage<JobData>
                 .retry { throwable ->
                     putJobData.loggingService().logMessage("Error checking status of job " + jobUUID() + " will retry", LogType.ERROR)
                     LOG.error("Unable to check status of job " + jobUUID(), throwable)
-                    true
+                    when (throwable) {
+                        is IllegalStateException -> false
```

Comment:
Can we look into seeing if we can improve the exception we key off of here.  On first inspection it's unclear what the IllegalStateException means here.